src/ucs/arch/x86_64/cpu.c

```diff
@@ -707,6 +736,319 @@ ucs_status_t ucs_arch_get_cache_size(size_t *cache_sizes)
     return cache_count == UCS_CPU_CACHE_LAST ? UCS_OK : UCS_ERR_UNSUPPORTED;
 }
 
+#if ((ENABLE_NT_BUFFER_TRANSFER) && (__AVX__))
+
+#define SWITCH_TO_NTSTORE_SZ (1464)
```

Comment:
can we make it "const size_t ..." inside ucs_x86_nt_dst_buffer_transfer?

Reply:
Done.